### PR TITLE
Update README and unity doc

### LINKED ISSUE #1204: Update README + Unity Documentation
- Update README doc for `AuthCallback` and `CreateTokenRequestAsync`.
- Update README for the latest supported platforms, including unity.
- Update beta specification for the library.
- Update MAUI support for the library.

### PR DESCRIPTION
**Pull request by sacOO7**
Fixed #1204

### CODE REVIEW

File: unity/README.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    }
+```
+- Please take a look at [Unity Demo Chat App](./Assets/Ably/Examples/Chat/) for understanding ably SDK setup.
+
+### Unsupported Platforms
+- WebGL: Due to incompatibility with Websockets.<br/>
+Read [Direct Socket Access](https://docs.unity3d.com/2019.3/Documentation/Manual/webgl-networking.html) section under WebGL Networking.
+
+### Contributing
+- Please take a look at [contributing doc](CONTRIBUTING.md) for information related to local dev-setup, writing and running tests.
```
Comment (tomczoink) [inline unity/README.md:81]:
```suggestion
- Please take a look at the [contributing documentation](CONTRIBUTING.md) for information relating to local development setup, and writing and running tests.
```

File: unity/README.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+- Detailed information related to updating ably DLLs using [unity plug-ins](https://docs.unity3d.com/Manual/Plugins.html) is given at [updating-ably-unitypackage](CONTRIBUTING.md#updating-ably-unitypackage).
```
Comment (tomczoink) [inline unity/README.md:82]:
```suggestion
- Detailed information related to updating Ably DLLs using [Unity Plugins](https://docs.unity3d.com/Manual/Plugins.html) is available in [updating-ably-unitypackage](CONTRIBUTING.md#updating-ably-unitypackage).
```

**Review by tomczoink — CHANGES_REQUESTED** [review]
Technically the recommended steps seem to work for me, just a few grammatical and formatting changes I think might help.

File: unity/README.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  * Api Compatibility Level should be '.NET Standard 2.0'
+
+### Downloading Unity Package
+- Please download the latest Unity package from the [GitHub releases page](https://github.com/ably/ably-dotnet/releases/latest). All releases from 1.2.4 has `.unitypackage` included.
+
+### Importing Unity Package
+- Import package by going to Assets -> Import Package -> Custom Package.
+  For detailed information, visit https://docs.unity3d.com/Manual/AssetPackagesImport.html
+- Make sure to [disable assembly validation](CONTRIBUTING.md#disable-assembly-validation-error) if it fails due to conflict with internal newtonsoft json library.
+- Please set `ClientOptions.AutomaticNetworkStateMonitoring` to `false` in the code, since the feature is not supported and throws runtime exception.
```
[the remark this answers is not in the document]
Comment (sacOO7) [reply unity/README.md:25]: We are planning to fix it here https://github.com/ably/ably-dotnet/issues/1212

**Review by tomczoink — APPROVED** [review]
LGTM

**Pull request merged**